packages/react-storybook-addon/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export type { FluentGlobals, FluentStoryContext } from './hooks';
-export { themes, ThemeIds } from './theme';
+export { themes } from './theme';
+export type { ThemeIds } from './theme';
```

Comment:
Otherwise it fails 🤒

Reply:
lets wait for https://github.com/microsoft/fluentui/pull/20797 . My bad I should explicitly mention that this package should be migrated after all dependant packages